### ISSUE TITLE
altitude_relative in ALTITUDE in not altitude above home position, but home altitude

### ISSUE THREAD

**Issue by svpcom** [PX4/Firmware#4505]
In mavlink msg definition:
```
float altitude_relative; /*< This is the altitude above the home position. It resets on each change of the current home position.*/
```
but in ``src/modules/mavlink/mavlink_messages.cpp``:
```
msg.altitude_relative = (_home_time > 0) ? home.alt : 0.0f / 0.0f;
```
it seems it should be:
```
msg.altitude_relative = (_home_time > 0) ? (pos.alt - home.alt) : 0.0f / 0.0f;
```

**Comment by svpcom** [comment]
@LorenzMeier FYI

**Comment by LorenzMeier** [comment]
Fixed in 34baf01.
